### PR TITLE
Make testsuite pass on RHEL 7.1

### PR DESCRIPTION
**Pull request by ctomc**

### CODE REVIEW

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jca/datasource/remove/EjbAppDsRemoveTestCase.java [hunk 1]
```diff
@@ -54,16 +54,11 @@ public static JavaArchive deploy() {
...
     public void testDatasourceRemove() throws Exception {
         final RemoveDsBeanRemote bean = lookupEJB(MODULE_NAME, RemoveDsBean.class, RemoveDsBeanRemote.class);
         bean.testDatasource();
-        try {
-            remove(datasourceAddress());
-            fail("Removal of the datasource should have failed.");
-        } catch (MgmtOperationException expected) {
-        }
-        bean.testDatasource();
+        remove(datasourceAddress());
```
Comment (bstansberry) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jca/datasource/remove/EjbAppDsRemoveTestCase.java]: This changes the test be removing L66.

Comment (ctomc) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jca/datasource/remove/EjbAppDsRemoveTestCase.java]: tnx, that was oversight

**Comment by bstansberry** [comment]
retest this please

**Comment by ctomc** [comment]
last run ^^ was on Centos 7.1 (build00-el7) build agent.

**Comment by bstansberry** [comment]
This had problems on master-ignore on Windows:

http://brontes.lab.eng.brq.redhat.com/viewLog.html?buildId=50174&tab=buildResultsDiv&buildTypeId=WF_MasterIgnoreWindows

**Comment by ctomc** [comment]
@bstansberry strange, I rebased this PR to master-ignore and ran jobs ^^ all passed.
also second run of master-ignore on windows passed as well :|

**Pull request merged**